pkg/api/testing/fuzzer.go

```diff
@@ -315,7 +315,7 @@ func FuzzerFor(t *testing.T, version string, src rand.Source) *fuzz.Fuzzer {
 		},
 		func(e *api.Event, c fuzz.Continue) {
 			c.FuzzNoCustom(e) // fuzz self without calling this function again
-			// Fix event count to 1, otherwise, if a v1beta1 or v1beta2 event has a count set arbitrarily, it's count is ignored
+			// Fix event count to 1, otherwise, if a v1beta1 or v1beta2(deleted on June 1, 2015) event has a count set arbitrarily, it's count is ignored
```

Comment:
Comment is confusing, just remove entirely.

Also, if I understand the comment, it's safe to remove the fuzz function (as long as the tests still pass that way).